register/views.py

```diff
@@ -43,6 +45,7 @@ def register_user(request, user_type):
             if form is not None and form.is_valid():
                 password = form.cleaned_data["input_password"]
                 f = form.save(commit=False)
+                f.school = HighSchool.objects.get(id=1)
```

Comment:
Add a TODO mentioning updating this logic to get the school from the form itself and not hard code (as mentioned in the comments above for "register/forms.py")